boards/remote/Makefile.include

```diff
@@ -1,10 +1,10 @@
-# define the cpu used by the Re-mote board
+# define the cpu used by the RE-mote board
 export CPU       =  cc2538
-export CPU_MODEL = cc2538sf53
+export CPU_MODEL =  cc2538sf53
 
 # define the default port depending on the host OS
-PORT_LINUX  ?= /dev/ttyUSB1
-PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
+PORT_LINUX  ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(shell ls -1 /dev/tty.usbserial-* | head -n 2 | tail -n 1)
 
```

Comment:
I have a little question here... some PR already "fixes" this line to use the previous one. See #4913